crates/ra_ide/Cargo.toml

```diff
@@ -17,6 +17,12 @@ indexmap = "1.3.2"
 itertools = "0.9.0"
 log = "0.4.8"
 rustc-hash = "1.1.0"
+rand = { version = "0.7.3", features = ["small_rng"] }
```

Comment:
We use `oorandom` rather than `rand`. But, more generally, why would one need random numbers to resolve links?

Reply:
Just an error in the merge I think.